### PR TITLE
Update HTML to HTML5

### PR DESCRIPTION
**Pull request by stephaniehobson**
Updates to HTML to reflect better support for HTML5 elements or HTML5 syntax.

## Significant changes and points to review

- Remove trailing slash from empty elements
- Remove type and char declaration from `script`
- Remove roles from `main` and `article` elements
- Found a random `</li>` with no opening one

## Issue / Bugzilla link

n/a

### CODE REVIEW

File: bedrock/base/templates/base-protocol.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
     <!--[if lt IE 9]>
       {# Only needed for IE before v9 #}
-      <script type="text/javascript" src="{{ static('js/ie/libs/html5shiv.js') }}"></script>
+      <script src="{{ static('js/ie/libs/html5shiv.js') }}"></script>
```
Comment (nathan-barrett) [inline bedrock/base/templates/base-protocol.html:56]: If this script is for IE <= 9, do we need to keep the type attr? I'm not sure on the answer!

**Review by nathan-barrett — APPROVED** [review]
nice work! so long `role=main`! Wondering if we should push this to the run-integration-tests branch before we merge, only since this touches so many files

File: bedrock/base/templates/base-protocol.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/base/templates/base-protocol.html:56]: Seems like no. It appears that it never did anything for the actual browser parsers 😝 (probably why it isn't necessary anymore)

**Comment by pmac** [comment]
Not sure why tests didn't run against this PR, but this broke some tests looking for specific markup.

**Comment by pmac** [comment]
see https://app.circleci.com/pipelines/github/mozilla/bedrock/12006/workflows/8fd46644-da89-420a-aa6a-c7c1744b2644/jobs/50465

**Pull request merged**